vmdb/app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -51,7 +51,16 @@ def save_container_groups_inventory(ems, hashes, target = nil)
                 []
               end
 
-    save_inventory_multi(:container_groups, ems, hashes, deletes, [:ems_ref], [:container_definitions, :containers, :labels])
+    hashes.each do |h|
+      next if h[:container_node_name].nil?
+      node = ContainerNode.first(
+        :conditions => {:ems_id => ems.id, :name => h[:container_node_name]})
+      h[:container_node_id] = node.nil? ? nil : node.id
```

Comment:
@Fryguy should I use something like:

    hashes.each do |h|
      h[:container_node_id] = h.fetch_path(:container_node, :id)
    end

on the saving side as well? It works but I am not sure if I am supposed do that (it resemble the method I was using before, but maybe it's saving a db query?)